lib/watir/wait.rb

```diff
@@ -246,5 +246,23 @@ def wait_while_present(timeout = nil)
       # it's not present
     end
 
+    #
+    # Waits until the element is stale.
+    #
+    # @example
+    #   element = browser.text_field(name: "abrakadbra").wait_until_stale
+    #
+    # @param [Fixnum] timeout seconds to wait before timing out
+    #
+    # @see Watir::Wait
+    # @see Watir::Element#stale?
+    #
+
+    def wait_until_stale(timeout = nil)
```

Comment:
wait_while_stale makes no sense, because once an element goes stale it never comes back.

This is the only reason I can come up with to want to use #always_locate - when you are taking an action on an element until it goes stale and rescuing the exception as part of flow control.